modules/atomic-widgets/prop-types/concerns/has-atomic-base.php

```diff
@@ -68,30 +68,33 @@ private static function validate_schema( array $schema ) {
 		}
 	}
 
-	private function sanitize_atomic_styles( array $styles ) {
-		foreach ( $styles as $style ) {
-			[$is_valid, $sanitized, $errors_bag] = Style_Validator::make( Style_Schema::get() )->validate( $style );
+	private function sanitize_atomic_styles( array $styles ): array {
```

Comment:
Maybe we should change the name for `parse_atomic_styles?`

Reply:
Can we use the `parse` method?

Reply:
Following our discussion - changed to parse, moved all errors to be inside the props_parser->parse method

(in other words - done 😝)